airflow/models/dag.py

```diff
@@ -2379,6 +2413,8 @@ class DagModel(Base):
         Index('idx_next_dagrun_create_after', next_dagrun_create_after, unique=False),
     )
 
+    parent_dag = relationship("DagModel", remote_side=[dag_id])
```

Comment:
This relationship was added for "Duck-type compatibility" between a DAG and a DagModel in the case of subdags -- both now have a "parent_dag" attribute.